src/span-data.ts

```diff
@@ -18,19 +18,19 @@
 
 declare global {
   interface CallSite {
```

Comment:
Could this type definition go in the node type definitions?

Reply:
Done